notebooks/catvton/README.md

```diff
@@ -0,0 +1,29 @@
+# Virtual Try-On with CatVTON and OpenVINO
+
+### Abstract
```

Comment:
for me it is slightly strange to start notebook and readme with abstract subarticle, that is stylistically different from other notebooks. Could you please remove header in both readme and notebook?